moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -127,22 +191,22 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
   {
     if (!(trajectory.fillInFromTrajectory(*res.trajectory_[0])))
     {
-      ROS_ERROR_STREAM_NAMED("chomp_planner", "Input trajectory has less than 2 points, "
-                                              "trajectory must contain at least start and goal state");
+      RCLCPP_ERROR(LOGGER, "Input trajectory has less than 2 points, "
+                           "trajectory must contain at least start and goal state");
       return false;
     }
   }
   else
   {
-    ROS_ERROR_STREAM_NAMED("chomp_planner", "invalid interpolation method specified in the chomp_planner file");
+    RCLCPP_ERROR(LOGGER, "invalid interpolation method specified in the chomp_planner file");
     return false;
   }
 
-  ROS_INFO_NAMED("chomp_planner", "CHOMP trajectory initialized using method: %s ",
-                 (params.trajectory_initialization_method_).c_str());
+  RCLCPP_INFO(LOGGER, "CHOMP trajectory initialized using method: %s ",
+              (params.trajectory_initialization_method_).c_str());
 
   // optimize!
-  ros::WallTime create_time = ros::WallTime::now();
+  std::chrono::time_point<std::chrono::system_clock> create_time = std::chrono::system_clock::now();
```

Comment:
Using `chrono` directly is fine with me. `rclcpp::Duration` could make the code below a bit more readable and make it easier to access the seconds as double.